backend-rust/README.md

```diff
@@ -91,3 +156,26 @@ This will create two files in the directory:
 - `<database-version>_<description>.up.sql` for the SQL code to bring the database up from the previous version.
 - `<database-version>_<description>.down.sql` for the SQL code reverting back to the previous version.
 
+
+### Database deletion
+
+If you started the database with the `docker-compose` file and you want to restart the database with fresh data,
+delete the data in the `data` folder with the command:
+
+```
+sudo rm -r data/
+```
```

Comment:
I updated it to use the `--force` flag mainly because I have other services running linked to the database usually as well.

```
sqlx database reset --force
```